src/app/pages/LiveRadioPage/index.jsx

```diff
@@ -95,10 +95,10 @@ const LiveRadioPage = ({ pageData }) => {
     translations,
   );
 
-  const { enabled } = useToggle('liveRadioSchedule');
-  // onLiveRadioPage is enabled on Persian to render the schedule for bbc_dari_radio
-  // however bbc_persian_radio should not show the schedule
-  const showSchedule = enabled && radioScheduleOnPage;
+  const { enabled, value } = useToggle('liveRadioSchedule');
+
+  const showSchedule =
+    enabled && RegExp(value).test(service) && radioScheduleOnPage;
```

Comment:
Minor performance gains from not constructing and testing the regex when `showSchedule` will end up being `false` anyway
```suggestion
  const showSchedule =
    radioScheduleOnPage && enabled && RegExp(value).test(service);
```